S/SortingAlgorithms/Compat.toml

```
@@ -1,7 +1,11 @@
 ["0-0.3.1"]
 DataStructures = "0.9.0-0.18"
-julia = ["0.7", "1"]
+julia = ["0.7", "1-1.8"]
 
-["0.3.2-1"]
+["0.3.2-0"]
```

```suggestion
["0.3.2-1.0"]
```
I think the current change fails to specify anything for versions 1.0.0 and 1.0.1.